practices/automateRepeatableTasks/readme.md

```diff
@@ -0,0 +1,65 @@
+# Automate Repeatable Tasks
+
+Humans are error prone; we are very bad at doing the same thing over and over. Machines, on the other hand, are very good at repeatable tasks! When working on a software project, we should strive to reduce any manual steps as much as possible. This will save time, reduce errors, but also add predictability to our software. "Works on my machine" is never an acceptable excuse! =)
+
+## The What and the Why
+
+### The What
+
+Automation of repeatable task is pretty self explanatory: the process of implementing repeatable tasks automatically. Repeatable task can be found in many aspects of work at Concord such as planning, development and deployment. Depending on the type of task it may require different methods for automation.
+
+### The Why
+
+Automation of repeatable tasks reduces repeatability errors created by humans and increases efficiency.
+
+## Examples
+
+### Planning
+
+Some examples of tasks you may need to automate in planning could be:
+
+1. Backlog, Sprint and Retro set ups
+2. Weekly stand up meetings
+
+Solutions:
```

Comment:
I know this is under the examples heading, but I think it's still useful to call these out as potential solutions

Reply:
I'm not sure I understand what you mean here. Are you saying that Backlog, Sprint and Retro set ups would be solutions to repeatable tasks while managing a project?

Reply:
I'm just suggesting that line 24 literally be updated to say "Potential Solutions:" instead (and the similar lines below)

Reply:
updated!